test/sagemaker_tests/huggingface/inference/integration/sagemaker/test_neuron_hosting.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+import pytest
+import sagemaker
+from sagemaker.model import Model
+from sagemaker.predictor import Predictor
+from sagemaker.serializers import JSONSerializer
+from sagemaker.deserializers import JSONDeserializer
+
+
+from ...integration import model_dir, pt_neuron_model,pt_neuron_script
+from ...integration.sagemaker.timeout import timeout_and_delete_endpoint
+from .... import invoke_pytorch_helper_function
+
+
+@pytest.mark.model("tiny-distilbert")
+@pytest.mark.processor("neuron")
+@pytest.mark.neuron_test
```

Comment:
in order to use this marker you'll need to update conftest to do something with this marker

Reply:
I thought it existed already. I basically copy-pasted the contest from PyTorch for neuron